### DISCUSSION TITLE
[Feature] can skywalking-kubernetes  update for postgresql etc.

### DISCUSSION THREAD

**Discussion by mypublicuse** [apache/skywalking#9448] [Ideas]
### Search before asking

- [X] I had searched in the [issues](https://github.com/apache/skywalking/issues?q=is%3Aissue) and found no similar feature requirement.

### Description

I read the skywalking-kubernetes .but I not found the values for  postgresql!  
can the helm chart  upgrade ,add the example for  postgresql?

### Are you willing to submit a PR?

- [ ] Yes I am willing to submit a PR!

### Code of Conduct

- [X] I agree to follow this project's [Code of Conduct](https://www.apache.org/foundation/policies/conduct)

**Comment by wu-sheng** [comment]
The storage option could be activated through system env, the helm has opened for that.
